assets/src/components/customers/CustomersTable.tsx

```diff
@@ -4,7 +4,7 @@ import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc';
 import {Customer} from '../../types';
 import {Badge, Button, Table, Text, Tooltip} from '../common';
-import CustomerDetailsModal from './CustomerDetailsModal';
+import CustomerDetailsModal from '../customers/CustomerDetailsModal';
```

Comment:
was this change necessary? 🤔

Reply:
(on an unrelated note, it might be interesting to switch to using absolute(?) paths rather than relative paths)

Reply:
No, I'm not sure why this changed. I must have moved some stuff around and accidentally made this change. I'll fix it. 

re: absolute paths - I don't have an opinion either way.